wbooks/src/app/screens/BookCard/styles.ts

```diff
@@ -0,0 +1,31 @@
+import { StyleSheet } from 'react-native';
+import { white } from '@constants/colors';
+
+export default StyleSheet.create({
+  wrapper: {
+    backgroundColor: white,
+    borderRadius: 5,
+    flexDirection: 'row',
+    marginTop: 17,
+    height: 90,
+    width: 335
```

Comment:
No, never use static values for components (only for Images or specific cases when you can't use margin or padding).
Use padding in this case, check Marvel again